src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/StashBuildListener.java

```diff
@@ -61,7 +64,17 @@ public void onCompleted(AbstractBuild<?, ?> abstractBuild, @Nonnull TaskListener
     } else {
       buildUrl = rootUrl + abstractBuild.getUrl();
     }
-    repository.deletePullRequestComment(cause.getPullRequestId(), cause.getBuildStartCommentId());
+
+    try {
+      repository.deletePullRequestComment(cause.getPullRequestId(), cause.getBuildStartCommentId());
+    } catch (StashApiException e) {
+      String logmsg =
+          format(
+              "%s: cannot delete Build Start comment for pull request %s: %s",
+              abstractBuild.getParent().getDisplayName(), cause.getPullRequestId(), e);
+      logger.info(logmsg);
+      listener.getLogger().println(logmsg);
```

Comment:
I think logging to the build output is enough
I would only log to global logger if there is no build to log to

Reply:
OK, fixed. Good point.